README.md

```diff
@@ -82,6 +82,15 @@ cookiecutter gh:elixir-cloud-aai/cookiecutter-python
 > `update-template` job from `update.yaml` github action workflow file as
 > `cookiecutter` doesn't support autosync like Cruft.
 
+### Extra dependency
+
+Some checks and configuration might require additional dependencies.
+While we try to keep the template as minimal as possible and have a fallback
+option to use the default configuration, we recommend installing the following:
```

Comment:
I don't really understand this. Why not simply include them in your managed dependencies? Or otherwise at least include them in some optional section of `pyproject.toml`. Like this, it's very easy to overlook this.

Reply:
These deps aren't required for generated project but rather for templating them.

Reply:
I understand that, but you'd still want to provide them in some conventional file, like `requirements.txt`, `pyproject.toml` or similar, no?

Reply:
removed all the deps :)